packages/stories-utils/src/__tests__/prepareStory.test.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import prepareStory from '../prepareStory';
+
+test('no args', () => {
+  const Story = () => <button>brew</button>;
+
+  render(prepareStory(Story));
+  expect(screen.getByRole('button', { name: 'brew' })).toBeTruthy();
```

Comment:
We could've tested this differently, but I decided to make a legit story and actually render it.